R/std_glm.R

```diff
@@ -0,0 +1,38 @@
+#' Standardized Generalized Linear Models
+#'
+#' @description `r lifecycle::badge("experimental")`
+#'
+#' @param formula (`formula`)\cr the model formula. Must include `trt` to indicate the treatment variable.
+#' @param data (`data.frame`)\cr the dataset to be used.
+#' @param trt (`string`)\cr the treatment variable.
+#' @param ... additional arguments for `glm()`.
+#'
+#' @note Response must be binary.
+#' @export
+#'
+#' @examples
+#' std_glm(Sepal.Length ~ Species, data = iris, trt = "Species")
+std_glm <- function(formula, data, trt, ...) {
+  assert_formula(formula)
+  assert_string(trt)
+  assert_subset(trt, all.vars(formula[[length(formula)]]))
```

Comment:
Minor comment: I noticed assertion does not pass if using special operator in formula, e.g. `y ~ .` (useful if users want to include all covariates)